sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detections/board_detection.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         self.rows = rows
         self.cols = cols
 
+        # cSpell:enableCompoundWords
         self._precomputed_single_shot_model = None
```

Comment:
We can add `precomputed` to the dictionary.

Reply:
added it to the respective PR

Reply:
The respective PR? What is it? :thinking: 
This one? https://github.com/tier4/autoware-spell-check-dict/pull/456

Reply:
By the way, I've already added several words to the dictionary.
Please confirm https://github.com/tier4/autoware-spell-check-dict/pulls?q=is%3Apr+is%3Aclosed.

![image](https://user-images.githubusercontent.com/31987104/217228661-13651371-2552-4292-b2d1-63aef621b468.png)